### PR TITLE
update news and description to 0.6.10, [skip vbump]

### PR DESCRIPTION
**Pull request by shajoezhu**
bloced by #932

### CODE REVIEW

File: NEWS.md [hunk 1]
```diff
@@ -1,7 +1,12 @@
...
+### New Features
+ * Added top left information handling (now bold and bottom aligned).
+ * Added `section_properties_default()` function to define standard portrait properties for tables.
+
+### Enhancements
  * Modified `reorder_split_levels()` to cover more edge cases and be more stringent in the allowed inputs.
- * Removed `tt` input from `theme_docx_default()` and added internal handling for row classes and number of columns.
+ * Removed table tree `tt` input from `theme_docx_default()` and added code to handle row classes and number of columns internally.
  * Reworked padding and spacing in default theme `theme_docx_default()`.
  * Added top left information handling (now bold and bottom aligned).
```
Comment (shajoezhu) [inline NEWS.md:11]:
```suggestion
```

File: NEWS.md [hunk 2]
```diff
@@ -1,9 +1,13 @@
...
+ * Added top left information handling (now bold and bottom aligned).
+ * Added `section_properties_default()` function to define standard portrait properties for tables.
+
+### Enhancements
  * Modified `reorder_split_levels()` to cover more edge cases and be more stringent in the allowed inputs.
- * Removed `tt` input from `theme_docx_default()` and added internal handling for row classes and number of columns.
+ * Removed table tree `tt` input from `theme_docx_default()` and added code to handle row classes and number of columns internally.
  * Reworked padding and spacing in default theme `theme_docx_default()`.
- * Added top left information handling (now bold and bottom aligned).
  * Now users can add more than one theme to `tt_to_flextable()`, and/or extend themes.
```
Comment (Melkiades) [inline NEWS.md:11]: feature too?

File: NEWS.md [hunk 3]
```diff
@@ -1,9 +1,13 @@
...
  * Added default theme for `.html` outputs.
  * Added parameter `bold_titles` to `tt_to_flextable()` to bold titles.
```
Comment (Melkiades) [inline NEWS.md:13]: feature maybe?

File: NEWS.md [hunk 4]
```diff
@@ -1,9 +1,13 @@
-## rtables 0.6.9.9016
-### Features
+## rtables 0.6.10
+
+### New Features
+ * Added top left information handling (now bold and bottom aligned).
+ * Added `section_properties_default()` function to define standard portrait properties for tables.
+
```
Comment (shajoezhu) [inline NEWS.md:6]:
```suggestion
 * Added default theme for `.html` outputs.
 * Added parameter `bold_titles` to `tt_to_flextable()` to bold titles.
 
```

File: NEWS.md [hunk 5]
```diff
@@ -1,9 +1,15 @@
...
+ * Added default theme for `.html` outputs.
+ * Added parameter `bold_titles` to `tt_to_flextable()` to bold titles.
+ 
+### Enhancements
  * Modified `reorder_split_levels()` to cover more edge cases and be more stringent in the allowed inputs.
- * Removed `tt` input from `theme_docx_default()` and added internal handling for row classes and number of columns.
+ * Removed table tree `tt` input from `theme_docx_default()` and added code to handle row classes and number of columns internally.
  * Reworked padding and spacing in default theme `theme_docx_default()`.
- * Added top left information handling (now bold and bottom aligned).
  * Now users can add more than one theme to `tt_to_flextable()`, and/or extend themes.
```
Comment (shajoezhu) [inline NEWS.md:13]:
```suggestion
```

File: NEWS.md [hunk 6]
```diff
@@ -1,9 +1,15 @@
-## rtables 0.6.9.9016
-### Features
+## rtables 0.6.10
+
+### New Features
+ * Added top left information handling (now bold and bottom aligned).
+ * Added `section_properties_default()` function to define standard portrait properties for tables.
+ * Added default theme for `.html` outputs.
+ * Added parameter `bold_titles` to `tt_to_flextable()` to bold titles.
```
Comment (shajoezhu) [inline NEWS.md:7]:
```suggestion
 * Added parameter `bold_titles` to `tt_to_flextable()` to bold titles.
  * Now users can add more than one theme to `tt_to_flextable()`, and/or extend themes.
```

File: NEWS.md [hunk 7]
```diff
@@ -1,9 +1,15 @@
...
+ 
+### Enhancements
  * Modified `reorder_split_levels()` to cover more edge cases and be more stringent in the allowed inputs.
- * Removed `tt` input from `theme_docx_default()` and added internal handling for row classes and number of columns.
+ * Removed table tree `tt` input from `theme_docx_default()` and added code to handle row classes and number of columns internally.
  * Reworked padding and spacing in default theme `theme_docx_default()`.
- * Added top left information handling (now bold and bottom aligned).
  * Now users can add more than one theme to `tt_to_flextable()`, and/or extend themes.
  * Added default theme for `.html` outputs.
```
Comment (shajoezhu) [inline NEWS.md:14]:
```suggestion
```

File: NEWS.md [hunk 8]
```diff
@@ -1,12 +1,17 @@
...
- * Reworked padding and spacing in default theme `theme_docx_default()`.
+## rtables 0.6.10
+
+### New Features
  * Added top left information handling (now bold and bottom aligned).
- * Now users can add more than one theme to `tt_to_flextable()`, and/or extend themes.
+ * Added `section_properties_default()` function to define standard portrait properties for tables.
  * Added default theme for `.html` outputs.
  * Added parameter `bold_titles` to `tt_to_flextable()` to bold titles.
+  * Now users can add more than one theme to `tt_to_flextable()`, and/or extend themes.
```
Comment (shajoezhu) [inline NEWS.md:8]:
```suggestion
 * Now users can add more than one theme to `tt_to_flextable()`, and/or extend themes.
```

**Pull request merged**